handlers/handlers.go

```diff
@@ -37,75 +55,85 @@ func GetById(w http.ResponseWriter, r *http.Request) {
 	pokemonId := pathParams["pokemonId"]
 	w.Header().Set("Content-Type", "application/json")
 
-	file, err := os.Open("assets/pokemon.csv")
+	file, err := os.Open(FilePath)
 	if err != nil {
-		fmt.Println(err)
 		w.WriteHeader(http.StatusInternalServerError)
-		w.Write([]byte("An error occured while opening the file"))
+		w.Write([]byte("Error reading CSV file"))
+		return
 	}
 	reader := csv.NewReader(file)
-	records, _ := reader.ReadAll()
-
+	records, err := reader.ReadAll()
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		w.Write([]byte("CSV is invalid"))
+		return
+	}
 	for _, pokemon := range records {
 		pokemonMap[pokemon[0]] = pokemon[1]
 	}
 
 	if _, ok := pokemonMap[pokemonId]; ok {
-		response, _ := json.Marshal(pokemonMap[pokemonId])
+		result := make(map[string]string)
+		result["id"] = pokemonId
+		result["name"] = pokemonMap[pokemonId]
+		response, err := json.Marshal(result)
+		if err != nil {
+			w.WriteHeader(http.StatusInternalServerError)
+			w.Write([]byte("Error parsing result"))
+			return
+		}
 		w.WriteHeader(http.StatusOK)
 		w.Write(response)
 	} else {
 		w.WriteHeader(http.StatusNotFound)
-		w.Write([]byte("Resource Not Found"))
+		w.Write([]byte("Resource Not Found with id"))
 	}
 }
 
-func Post(w http.ResponseWriter, r *http.Request) {
-	w.Header().Set("Content-Type", "application/json")
-	w.WriteHeader(http.StatusCreated)
-	w.Write([]byte(`{"message": "post called"}`))
-}
+func GetBerries(w http.ResponseWriter, r *http.Request) {
 
-func Put(w http.ResponseWriter, r *http.Request) {
-	w.Header().Set("Content-Type", "application/json")
-	w.WriteHeader(http.StatusAccepted)
-	w.Write([]byte(`{"message": "put called"}`))
-}
+	apiResponse, err := http.Get("https://pokeapi.co/api/v2/berry")
```

Comment:
Remember to also validate different status codes that could return from this call.